libqtile/widget/sensors.py

```diff
@@ -43,8 +43,8 @@ class ThermalSensor(base.InLoopPollText):
     """
 
     defaults = [
+        ("format", "{temp:.1f}{unit}", "Display string formatting"),
```

Comment:
Can you add which fields are available here? e.g.  I see that `tag` is also an option but users won't know they can use this unless they look at the source.